org.codeberg.dnkl.foot.metainfo.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+
+  <id>org.codeberg.dnkl.foot</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+
+  <developer_name>dnkl</developer_name>
+  <name>foot</name>
+  <summary>The fast, lightweight and minimalistic Wayland terminal emulator.</summary>
+  <description>
+    <p>The fast, lightweight and minimalistic Wayland terminal emulator.</p>
+  </description>
```

Comment:
Repeating the summary in the description doesn't make it a great app store entry.